src/ens/SetName.svelte

```diff
@@ -44,7 +44,7 @@
 
     if (resolved && isAddressEqual(resolved, org.address)) {
       try {
-        if (utils.isSafe(org.owner, config)) {
+        if (await utils.isSafe(org.owner, config)) {
```

Comment:
Aha, damn, good catch. I wonder if the linter can be setup to catch conditionals with not-even-close-to-being-a-boolean values.